lib/spack/spack/test/cmd/gpg.py

```diff
@@ -48,6 +52,8 @@ def test_find_gpg(cmd_name, version, tmpdir, mock_gnupghome, monkeypatch):
         assert spack.util.gpg.GPGCONF is not None
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
This module contains a module-level skip and a skip for this test. I assume the latter can be removed.

Reply:
Yes. I will be going through all the tests looking for this and similar scenarios.